src/OSPSuite.Presentation/Presenters/ObservedData/DataRepositoryDataPresenter.cs

```diff
@@ -97,7 +60,7 @@ public void AddData(int rowIndex)
       public IEnumerable<string> GetCellValidationErrorMessages(int rowIndex, int columnIndex, string newValue)
```

Comment:
Why do we need this? This is for validation of new value and we can't edit. Remove

Reply:
unless this is used for something else

Reply:
true, this is not needed anymore